transport/components.py

```diff
@@ -0,0 +1,35 @@
+from raiden.messages import Message as RaidenMessage
+from raiden.transfer.identifiers import QueueIdentifier
+from raiden.utils import Address
+
+
+class Params:
```

Comment:
Personally i dont like the Params classname. 

We are at transport/components, therefore this say to me "the parameters is a component of a transport", but, the parameters are a component of the transport message

I think MessageParams is better.

Reply:
i agree, fixed